README.adoc

```diff
@@ -4002,6 +4002,30 @@ format('Hello, %{name}', name: 'John')
 format('Hello, %<name>s', name: 'John')
 ----
 
+=== Long Strings [[heredoc-long-strings]]
+
+Break long strings into multiple lines but don't concatenate them with `+`.
+If you want to add newlines, use `Heredoc`. Otherwise use `\`:
+
+[source,ruby]
+----
+# bad
+"Lorem Ipsum is simply dummy text of the printing and typesetting industry. " +
+"Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, " +
+"when an unknown printer took a galley of type and scrambled it to make a type specimen book."
+
+# good
+"Lorem Ipsum is simply dummy text of the printing and typesetting industry. "\
+"Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, "\
+"when an unknown printer took a galley of type and scrambled it to make a type specimen book."
+
+# good
+<<-LOREM
+Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+LOREM
```

Comment:
I think that this example indenting using squiggly heredoc is practical if there is no reason because Ruby 2.3 has been EOL.

```ruby
<<~LOREM
  Lorem Ipsum is simply dummy text of the printing and typesetting industry.
  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
  when an unknown printer took a galley of type and scrambled it to make a type specimen book.
LOREM
```

Reply:
Please add indent at beginning of line.

```diff
  <<~LOREM
- Lorem Ipsum is simply dummy text of the printing and typesetting industry.
- Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
- when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+   Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+   Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+   when an unknown printer took a galley of type and scrambled it to make a type specimen book.
  LOREM
```

Reply:
@koic Done.